iModelCore/BeSQLite/SQLite/blockcachevfsd.c

```diff
@@ -810,12 +810,13 @@ static int main_copy(int argc, char **argv){
   return rc;
 }
 
-static void filesCb(void *pApp, int rc, char *z){
+static int filesCb(void *pApp, int rc, char *z){
   if( rc==SQLITE_OK ){
     if( z ) printf("%s\n", z);
   }else{
     fatal_error("error (%d) - %s", rc, z);
   }
+  return rc;
```

Comment:
```suggestion
  return SQLITE_OK;
```

Reply:
Apparently I missed this one change last time I updated CloudSQLite, the change on trunk returns SQLITE_OK. So going to commit that.